src/Traits/PerformsSingleSignOn.php

```diff
@@ -63,4 +62,26 @@ public function getServiceProvider($request)
     {
         return base64_encode($request->getAssertionConsumerServiceURL());
     }
+
+    /**
+     * @return \LightSaml\Credential\X509Certificate
+     */
+    protected function getCertificate(): X509Certificate
+    {
+        $certificate = config('samlidp.cert') ?: Storage::disk('samlidp')->get(config('samlidp.certname', 'cert.pem'));
+
+        return (strpos($certificate, 'file://') === 0)
+            ? X509Certificate::fromFile($certificate)
+            : (new X509Certificate)->loadPem($certificate);
+    }
```

Comment:
I feel like this makes it a bit easier to read.
```suggestion
    protected function getCertificate(): X509Certificate
    {
        $certificate = config('samlidp.cert') ?: Storage::disk('samlidp')->get(config('samlidp.certname', 'cert.pem'));

        if (strpos($certificate, 'file://') === 0) {
            return X509Certificate::fromFile($certificate);
        }

        return (new X509Certificate)->loadPem($certificate);
    }
```